packages/components/header/_header-base.scss

```diff
@@ -344,6 +345,7 @@
   text-decoration: underline;
   white-space: nowrap;
   @include nhsuk-font(16);
+  @include nhsuk-link-style-white;
```

Comment:
Again, might need to see how this lands in the `header-breaking-changes` branch. That uses a new, separate mixin for link styles as they need to inherit the text colour used in the white and blue header variants.